tests/module_api/test_api.py

```diff
@@ -635,6 +635,14 @@ def test_check_push_rules_actions(self) -> None:
             [{"set_tweak": "sound", "value": "default"}]
         )
 
+    def test_normalize_username(self) -> None:
+        username = "Haxxor"
+        username2 = "_leet"
+        username3 = "aNoThErTeSt"
+        self.assertEqual(self.module_api.normalize_username(username), "haxxor")
+        self.assertEqual(self.module_api.normalize_username(username2), "=5fleet")
+        self.assertEqual(self.module_api.normalize_username(username3), "anothertest")
+
```

Comment:
Given that we're re-exposing one function withouot any I'm not sure how much this test gives us. I suppose it does force us to check that we continue to expose this in the module API!

Reply:
You know I wasn't so sure of the value of these either but as I junior I try to adhere to a "test everything if possible" philosophy. Happy to dump them if that's cleaner.